submission/submission-webapp/src/itest/kotlin/ac/uk/ebi/biostd/itest/test/submission/submit/FileListValidationTest.kt

```diff
@@ -72,13 +82,94 @@ class FileListValidationTest(
 
     @Test
     fun `valid file list`() {
-        val fileListFile = tempFolder.createFile("Plate1.tif")
-        val fileList = tempFolder.createFile("FileList.json", getFileListContent().toString())
+        val previousVersion = tsv {
+            line("Submission", "S-FLV123")
+            line()
 
-        webClient.uploadFiles(listOf(fileListFile, fileList))
-        webClient.validateFileList(fileList.name)
+            line("Study")
+            line()
 
-        webClient.deleteFile(fileListFile.name)
+            line("File", "Plate2.tif")
+            line()
+        }.toString()
+
+        val file1 = tempFolder.createFile("Plate1.tif", "content-1")
+        val file2 = tempFolder.createFile("Plate2.tif", "content-2")
+        val fileListContent = tsv {
+            line("Files", "Resource")
+            line("Plate1.tif", "USER_SPACE")
+            line("Plate2.tif", "SUBMISSION")
+            line()
+        }.toString()
+
+        val fileList = tempFolder.createFile("valid-file-list.tsv", fileListContent)
+
+        webClient.uploadFiles(listOf(file1, fileList))
+        webClient.submitSingle(previousVersion, TSV, SubmissionFilesConfig(listOf(file2)))
+
+        webClient.validateFileList(fileList.name, previousVersionAccNo = "S-FLV123")
+
+        webClient.deleteFile(file1.name)
+        webClient.deleteFile(fileList.name)
+    }
+
+    @Test
+    @EnabledIfSystemProperty(named = "enableFire", matches = "true")
+    fun `valid file list including a file from FIRE`() {
+        val previousVersion = tsv {
+            line("Submission", "S-FLV124")
+            line()
+
+            line("Study")
+            line()
+
+            line("File", "Plate5.tif")
+            line()
+        }.toString()
+
+        val file3 = tempFolder.createFile("Plate3.tif", "content-3")
```

Comment:
any reason to start with the file number 3? super small detail but I think test will be way simpler if file list have a single file and is in Fire. The simpler and more focus we design the scenarios the better. 

For a complex scenarios mixed, we have the allIncluded test  (please consider same comment for other test)